swarm/storage/mru/handler.go

```diff
@@ -163,18 +169,17 @@ func (h *Handler) chunkSize() int64 {
 func (h *Handler) New(ctx context.Context, request *Request) error {
 
 	// frequency 0 is invalid
-	if request.frequency == 0 {
+	if request.metadata.Frequency == 0 {
 		return NewError(ErrInvalidValue, "frequency cannot be 0 when creating a resource")
 	}
 
 	// make sure owner is set to something
-	var zeroAddr = common.Address{}
-	if request.ownerAddr == zeroAddr {
+	if request.metadata.Owner == zeroAddr {
```

Comment:
If you find that private key, please let me know so we can celebrate.

I put that check there to help the user know they hadn't set `ownerAddr` to something. A common mistake. 

Common, I mean in me, since I was the only one using this and making stupid mistakes, and then trying to figure out why my signatures were not valid.